README.md

```diff
@@ -49,6 +49,100 @@ This will set up an Electrode webapplication which will have 2 of the above 6 mo
   - [Electrode Confippet](https://github.com/electrode-io/electrode-confippet)
   - [Electrode Javascript Bundle Viewer](https://github.com/electrode-io/electrify)
 
+## Progressive Web App (PWA) features supported by the Electrode framework  
+#### 1. Offline first  
+  Offline first lets your app run without a network connection. At the same time it provides a great performance boost for repeat visit to your web site.
+  This is done with a service worker and by pre-caching your static assets as well as run time caching of react routes.  
```

Comment:
Maybe make this more general. In practice we will be using runtime caching for CDN assets and API calls on top of React routes:

> runtime caching of routes and external resources